### PR TITLE
Add prompt_toolkit and implement autosuggestion option. Add deps for black

### PR DESCRIPTION
**Pull request by TetianaFilonenko**
- Add autocomplete option with prompt_toolkit
- Customize command line
- add deps for black and prompt_toolkit

https://github.com/TetianaFilonenko/project-team-4/assets/5654369/aee54814-4f1f-4c82-9fbb-f8934a62fdc2

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+mypy-extensions==1.0.0
+packaging==24.0
+pathspec==0.12.1
+platformdirs==4.2.0
+prompt-toolkit==3.0.43
 python-dateutil==2.9.0.post0
 six==1.16.0
+tomli==2.0.1
+typing_extensions==4.10.0
+wcwidth==0.2.13
```
Comment (svilav) [inline requirements.txt]: where do all these reqs come from?

I created new venv with only these 2 and it works:
```
Faker==23.3.0
prompt_toolkit==3.0.43
```

Comment (svilav) [reply requirements.txt]: I use `pipreqs` lib to generate requirements.txt. It mostly does a good job

**Review by svilav — APPROVED** [review]
Awesome. I tried it and it looks cool. 
I would mention in `help` that `->` need to be used to autocomplete as it is not obvious.

File: requirements.txt (hunk 1 above)
Comment (TetianaFilonenko) [reply requirements.txt]: thanks for the feedback, let me try to fix those deps

**Review by svilav — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (TetianaFilonenko) [reply requirements.txt]: Q: Do you know if pipreqs needs to be in deps too?

Comment (svilav) [reply requirements.txt]: The person trying our app will not need it, so my guess is no. Plus it didn't put itself in reqs

**Pull request merged**